system/lib/fetch/emscripten_fetch.cpp

```diff
@@ -66,16 +68,58 @@ emscripten_fetch_t *emscripten_fetch(emscripten_fetch_attr_t *fetch_attr, const
 	}
 
 	emscripten_fetch_t *fetch = (emscripten_fetch_t *)malloc(sizeof(emscripten_fetch_t));
+    if ( !fetch ) return 0;
```

Comment:
The convention is to have parentheses hug the content inside, i.e. ` if (!fetch)`